code/modules/power/lighting/light.dm

```diff
@@ -313,17 +313,18 @@
 			to_chat(user, span_notice("You replace [light_object]."))
 		else
 			to_chat(user, span_notice("You insert [light_object]."))
+		if(length(light_object.reagents.reagent_list))
+			create_reagents(LIGHT_REAGENT_CAPACITY, SEALED_CONTAINER | TRANSPARENT)
+			light_object.reagents.copy_to(reagents, LIGHT_REAGENT_CAPACITY)
+			light_object.reagents.Destroy()
```

Comment:
NO NO NO BAD BAD BAD BAD DO NOT DO THIS. NEVER manually call `Destroy()` 

`QDEL_NULL(light_object.reagents)`.